docs/plugin/alias.md

```diff
@@ -1,3 +1,36 @@
 # Using `Alias`
 Since Docforge can construct the content bundle from multiple repository sources and there could be a need for additional names for certain files,
```

Comment:
The plugin's functionality is to "virtually move" content to another place. And `there could be a need for additional names for certain files,` is not technically correct as this is related to the typical use of aliases and not this "virual move". Therefore could this page only contain the content from `manifests.md` as it is a sufficient explanation?